### PR TITLE
Proxy Subscriptions

### LINKED ISSUE #1350: Update proxy entity example to be clear on use of subscriptions
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [X] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
Examples are clear in their attribute usage.

## Current Behavior
For proxy entities:

https://docs.sensu.io/sensu-go/5.5/reference/entities/#proxy-entities-managed

The example given has the `subscriptions` attribute listed. While this is valid and has some use cases, it can lead to confusion that this is the place to for a proxy entity to subscribe to a check.

However, the `check` and `agent` running the check on behalf of the proxy entity need to have the matching `subscription` for the check to run, then the proxy entity to be created.

## Possible Solution
- [ ] Add a note that the subscription inside a proxy entity is not used to have the entity subscribe to the check.
- [ ] Possibly add use cases for that proxy entity subscription (silencing, dashboard filtering)

### PR DESCRIPTION
**Pull request by Blue0ctober**
removed subscriptions from proxy agent and added section on info for proxy entity checks

## Description

Closes #1350

### CODE REVIEW

File: content/sensu-go/5.12/reference/entities.md [hunk 1]
```diff
@@ -151,6 +149,10 @@ spec:
 
 {{< /language-toggle >}}
 
+### Proxy entities checks
+
+Once a proxy entity is created, you can define which agents will run the check by configuring a subscription. See [Proxy Requests][11] for details on creating a proxy check for a proxy entity.
```
Comment (cwjohnston) [inline content/sensu-go/5.12/reference/entities.md]: As written this doesn't help the reader to connect proxy entities with proxy_requests check capability. Let's reword this to describe the connection and link to the relevant section of the check reference as well as to the "monitoring external resources" guide.

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.12/reference/entities.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.12/reference/entities.md]: Maybe something like this? I'm sure it's not perfect, but I thought it could be a good starting point. Everything is pulled from other docs.

Proxy entities allow Sensu to [monitor external resources][12] on systems or devices where a Sensu agent cannot be installed, like a network switch or a website. You can configure a check with a proxy entity name so that Sensu creates an entity representing the site and reports the status of the site under this entity.

After you create a proxy entity, you can define which agents will run the check by configuring a subscription. See [proxy requests][11] for details on creating a proxy check for a proxy entity.

File: content/sensu-go/5.12/reference/entities.md [hunk 2]
```diff
@@ -151,6 +149,12 @@ spec:
 
 {{< /language-toggle >}}
 
+### Proxy entities checks
+
+Proxy entities allow Sensu to [monitor external resources][12] on systems or devices where a Sensu agent cannot be installed, like a network switch, website or API endpoint. You can configure a check with a proxy entity name to associate the check results with that proxy entity. If on the first check result the proxy entity does not exist, Sensu will create the entity as a proxy entity.
```
Comment (hillaryfraley) [inline content/sensu-go/5.12/reference/entities.md]:
```suggestion
Proxy entities allow Sensu to [monitor external resources][12] on systems or devices where a Sensu agent cannot be installed, like a network switch, website, or API endpoint. You can configure a check with a proxy entity name to associate the check results with that proxy entity. On the first check result, if the proxy entity does not exist, Sensu will create the entity as a proxy entity.
```

File: content/sensu-go/5.12/reference/entities.md [hunk 3]
```diff
@@ -151,6 +149,12 @@ spec:
...
+
+After you create a proxy entity check, you can define which agents will run the check by configuring a subscription. See [proxy requests][11] for details on creating a proxy check for a proxy entity.
```
Comment (hillaryfraley) [inline content/sensu-go/5.12/reference/entities.md]:
```suggestion
After you create a proxy entity check, define which agents will run the check by configuring a subscription. See [proxy requests][11] for details on creating a proxy check for a proxy entity.
```

**Comment by hillaryfraley** [comment]
Just a couple more suggestions...

**Review by hillaryfraley — APPROVED** [review]
LGTM!

**Comment by hillaryfraley** [comment]
https://github.com/sensu/sensu-docs/pull/1876 replaces this PR -- I could not seem to resolve the conflicts in this PR.

**Comment by hillaryfraley** [comment]
Merged https://github.com/sensu/sensu-docs/pull/1876
